From ffe4eea2e3f19fc3587d66b15d2c1f9f26c95c77 Mon Sep 17 00:00:00 2001
From: Patrick Monette <pmonette@chromium.org>
Date: Thu, 14 Oct 2021 11:06:59 -0700
Subject: [PATCH] Reland "Move task-related files from base/ to base/task/"

This is a reland of 092c30c5fd4def5a0c63d6f3a8953bd07768dd44

The 2 problematic subrepos have been migrated and a couple late
addition of the old headers were migrated.

Original change's description:
> Move task-related files from base/ to base/task/
>
> The forward headers are updated to point to the new location, and the
> header guards were fixed.
>
> Bug: 1255932
> Change-Id: Id84a0dd71720efee6d222a0fc0a89c1de8c1d72c
> Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3214950
> Commit-Queue: Gabriel Charette <gab@chromium.org>
> Reviewed-by: Gabriel Charette <gab@chromium.org>
> Owners-Override: Gabriel Charette <gab@chromium.org>
> Cr-Commit-Position: refs/heads/main@{#930627}

Bug: 1255932
Change-Id: I62eb6a9e47b64bdd6f381cf27fe7f625740b2222
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3216953
Reviewed-by: Gabriel Charette <gab@chromium.org>
Commit-Queue: Gabriel Charette <gab@chromium.org>
Owners-Override: Gabriel Charette <gab@chromium.org>
Cr-Commit-Position: refs/heads/main@{#931601}


CrOS-Libchrome-Original-Commit: 39810e70da06baca176c0cdbc93164be1f980f59
---
 base/{ => task}/bind_post_task.h                          | 8 ++++----
 base/task/bind_post_task_forward.h                        | 2 +-
 base/{ => task}/bind_post_task_internal.h                 | 6 +++---
 base/{ => task}/bind_post_task_unittest.cc                | 0
 base/{ => task}/bind_post_task_unittest.nc                | 2 +-
 base/{ => task}/deferred_sequenced_task_runner.cc         | 0
 base/{ => task}/deferred_sequenced_task_runner.h          | 6 +++---
 base/task/deferred_sequenced_task_runner_forward.h        | 2 +-
 .../{ => task}/deferred_sequenced_task_runner_unittest.cc | 0
 .../{ => task}/post_task_and_reply_with_result_internal.h | 6 +++---
 .../post_task_and_reply_with_result_internal_forward.h    | 2 +-
 base/{ => task}/sequenced_task_runner.cc                  | 0
 base/{ => task}/sequenced_task_runner.h                   | 6 +++---
 base/task/sequenced_task_runner_forward.h                 | 2 +-
 base/{ => task}/sequenced_task_runner_helpers.h           | 6 +++---
 base/task/sequenced_task_runner_helpers_forward.h         | 2 +-
 base/{ => task}/sequenced_task_runner_unittest.cc         | 0
 base/{ => task}/single_thread_task_runner.h               | 6 +++---
 base/task/single_thread_task_runner_forward.h             | 2 +-
 base/{ => task}/task_runner.cc                            | 0
 base/{ => task}/task_runner.h                             | 6 +++---
 base/task/task_runner_forward.h                           | 2 +-
 base/{ => task}/task_runner_unittest.cc                   | 0
 base/{ => task}/task_runner_util.h                        | 6 +++---
 base/task/task_runner_util_forward.h                      | 2 +-
 base/{ => task}/task_runner_util_unittest.cc              | 0
 base/{ => task}/updateable_sequenced_task_runner.h        | 6 +++---
 base/task/updateable_sequenced_task_runner_forward.h      | 2 +-
 28 files changed, 41 insertions(+), 41 deletions(-)
 rename base/{ => task}/bind_post_task.h (96%)
 rename base/{ => task}/bind_post_task_internal.h (96%)
 rename base/{ => task}/bind_post_task_unittest.cc (100%)
 rename base/{ => task}/bind_post_task_unittest.nc (97%)
 rename base/{ => task}/deferred_sequenced_task_runner.cc (100%)
 rename base/{ => task}/deferred_sequenced_task_runner.h (95%)
 rename base/{ => task}/deferred_sequenced_task_runner_unittest.cc (100%)
 rename base/{ => task}/post_task_and_reply_with_result_internal.h (83%)
 rename base/{ => task}/sequenced_task_runner.cc (100%)
 rename base/{ => task}/sequenced_task_runner.h (98%)
 rename base/{ => task}/sequenced_task_runner_helpers.h (90%)
 rename base/{ => task}/sequenced_task_runner_unittest.cc (100%)
 rename base/{ => task}/single_thread_task_runner.h (89%)
 rename base/{ => task}/task_runner.cc (100%)
 rename base/{ => task}/task_runner.h (98%)
 rename base/{ => task}/task_runner_unittest.cc (100%)
 rename base/{ => task}/task_runner_util.h (93%)
 rename base/{ => task}/task_runner_util_unittest.cc (100%)
 rename base/{ => task}/updateable_sequenced_task_runner.h (84%)

diff --git a/base/bind_post_task.h b/base/task/bind_post_task.h
similarity index 96%
rename from base/bind_post_task.h
rename to base/task/bind_post_task.h
index cd6214e51d..d9352a7abf 100644
--- a/base/bind_post_task.h
+++ b/base/task/bind_post_task.h
@@ -2,18 +2,18 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef BASE_BIND_POST_TASK_H_
-#define BASE_BIND_POST_TASK_H_
+#ifndef BASE_TASK_BIND_POST_TASK_H_
+#define BASE_TASK_BIND_POST_TASK_H_
 
 #include <memory>
 #include <type_traits>
 #include <utility>
 
 #include "base/bind.h"
-#include "base/bind_post_task_internal.h"
 #include "base/callback.h"
 #include "base/location.h"
 #include "base/memory/ref_counted.h"
+#include "base/task/bind_post_task_internal.h"
 #include "base/task/task_runner_forward.h"
 
 // BindPostTask() is a helper function for binding a OnceCallback or
@@ -105,4 +105,4 @@ RepeatingCallback<void(Args...)> BindPostTask(
 
 }  // namespace base
 
-#endif  // BASE_BIND_POST_TASK_H_
+#endif  // BASE_TASK_BIND_POST_TASK_H_
diff --git a/base/task/bind_post_task_forward.h b/base/task/bind_post_task_forward.h
index 162154b5a9..bcb4bab43a 100644
--- a/base/task/bind_post_task_forward.h
+++ b/base/task/bind_post_task_forward.h
@@ -5,6 +5,6 @@
 #ifndef BASE_TASK_BIND_POST_TASK_FORWARD_H_
 #define BASE_TASK_BIND_POST_TASK_FORWARD_H_
 
-#include "base/bind_post_task.h"
+#include "base/task/bind_post_task.h"
 
 #endif  // BASE_TASK_BIND_POST_TASK_FORWARD_H_
diff --git a/base/bind_post_task_internal.h b/base/task/bind_post_task_internal.h
similarity index 96%
rename from base/bind_post_task_internal.h
rename to base/task/bind_post_task_internal.h
index f6b9f5cbac..38e9b2732b 100644
--- a/base/bind_post_task_internal.h
+++ b/base/task/bind_post_task_internal.h
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef BASE_BIND_POST_TASK_INTERNAL_H_
-#define BASE_BIND_POST_TASK_INTERNAL_H_
+#ifndef BASE_TASK_BIND_POST_TASK_INTERNAL_H_
+#define BASE_TASK_BIND_POST_TASK_INTERNAL_H_
 
 #include <utility>
 
@@ -94,4 +94,4 @@ class BindPostTaskTrampoline {
 }  // namespace internal
 }  // namespace base
 
-#endif  // BASE_BIND_POST_TASK_INTERNAL_H_
+#endif  // BASE_TASK_BIND_POST_TASK_INTERNAL_H_
diff --git a/base/bind_post_task_unittest.cc b/base/task/bind_post_task_unittest.cc
similarity index 100%
rename from base/bind_post_task_unittest.cc
rename to base/task/bind_post_task_unittest.cc
diff --git a/base/bind_post_task_unittest.nc b/base/task/bind_post_task_unittest.nc
similarity index 97%
rename from base/bind_post_task_unittest.nc
rename to base/task/bind_post_task_unittest.nc
index 3d390dfd8b..abb027e7fb 100644
--- a/base/bind_post_task_unittest.nc
+++ b/base/task/bind_post_task_unittest.nc
@@ -5,7 +5,7 @@
 // This is a "No Compile Test" suite.
 // http://dev.chromium.org/developers/testing/no-compile-tests
 
-#include "base/bind_post_task.h"
+#include "base/task/bind_post_task.h"
 
 #include "base/bind.h"
 #include "base/callback.h"
diff --git a/base/deferred_sequenced_task_runner.cc b/base/task/deferred_sequenced_task_runner.cc
similarity index 100%
rename from base/deferred_sequenced_task_runner.cc
rename to base/task/deferred_sequenced_task_runner.cc
diff --git a/base/deferred_sequenced_task_runner.h b/base/task/deferred_sequenced_task_runner.h
similarity index 95%
rename from base/deferred_sequenced_task_runner.h
rename to base/task/deferred_sequenced_task_runner.h
index bb05f7e37e..f2d0bac038 100644
--- a/base/deferred_sequenced_task_runner.h
+++ b/base/task/deferred_sequenced_task_runner.h
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef BASE_DEFERRED_SEQUENCED_TASK_RUNNER_H_
-#define BASE_DEFERRED_SEQUENCED_TASK_RUNNER_H_
+#ifndef BASE_TASK_DEFERRED_SEQUENCED_TASK_RUNNER_H_
+#define BASE_TASK_DEFERRED_SEQUENCED_TASK_RUNNER_H_
 
 #include <vector>
 
@@ -93,4 +93,4 @@ class BASE_EXPORT DeferredSequencedTaskRunner : public SequencedTaskRunner {
 
 }  // namespace base
 
-#endif  // BASE_DEFERRED_SEQUENCED_TASK_RUNNER_H_
+#endif  // BASE_TASK_DEFERRED_SEQUENCED_TASK_RUNNER_H_
diff --git a/base/task/deferred_sequenced_task_runner_forward.h b/base/task/deferred_sequenced_task_runner_forward.h
index 8e91c63230..2d9a8d86b1 100644
--- a/base/task/deferred_sequenced_task_runner_forward.h
+++ b/base/task/deferred_sequenced_task_runner_forward.h
@@ -5,6 +5,6 @@
 #ifndef BASE_TASK_DEFERRED_SEQUENCED_TASK_RUNNER_FORWARD_H_
 #define BASE_TASK_DEFERRED_SEQUENCED_TASK_RUNNER_FORWARD_H_
 
-#include "base/deferred_sequenced_task_runner.h"
+#include "base/task/deferred_sequenced_task_runner.h"
 
 #endif  // BASE_TASK_DEFERRED_SEQUENCED_TASK_RUNNER_FORWARD_H_
diff --git a/base/deferred_sequenced_task_runner_unittest.cc b/base/task/deferred_sequenced_task_runner_unittest.cc
similarity index 100%
rename from base/deferred_sequenced_task_runner_unittest.cc
rename to base/task/deferred_sequenced_task_runner_unittest.cc
diff --git a/base/post_task_and_reply_with_result_internal.h b/base/task/post_task_and_reply_with_result_internal.h
similarity index 83%
rename from base/post_task_and_reply_with_result_internal.h
rename to base/task/post_task_and_reply_with_result_internal.h
index 881f4bad8d..003c27cfd5 100644
--- a/base/post_task_and_reply_with_result_internal.h
+++ b/base/task/post_task_and_reply_with_result_internal.h
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef BASE_POST_TASK_AND_REPLY_WITH_RESULT_INTERNAL_H_
-#define BASE_POST_TASK_AND_REPLY_WITH_RESULT_INTERNAL_H_
+#ifndef BASE_TASK_POST_TASK_AND_REPLY_WITH_RESULT_INTERNAL_H_
+#define BASE_TASK_POST_TASK_AND_REPLY_WITH_RESULT_INTERNAL_H_
 
 #include <memory>
 #include <utility>
@@ -34,4 +34,4 @@ void ReplyAdapter(OnceCallback<void(ReplyArgType)> callback,
 
 }  // namespace base
 
-#endif  // BASE_POST_TASK_AND_REPLY_WITH_RESULT_INTERNAL_H_
+#endif  // BASE_TASK_POST_TASK_AND_REPLY_WITH_RESULT_INTERNAL_H_
diff --git a/base/task/post_task_and_reply_with_result_internal_forward.h b/base/task/post_task_and_reply_with_result_internal_forward.h
index f48e565b59..017224b66e 100644
--- a/base/task/post_task_and_reply_with_result_internal_forward.h
+++ b/base/task/post_task_and_reply_with_result_internal_forward.h
@@ -5,6 +5,6 @@
 #ifndef BASE_TASK_POST_TASK_AND_REPLY_WITH_RESULT_INTERNAL_FORWARD_H_
 #define BASE_TASK_POST_TASK_AND_REPLY_WITH_RESULT_INTERNAL_FORWARD_H_
 
-#include "base/post_task_and_reply_with_result_internal.h"
+#include "base/task/post_task_and_reply_with_result_internal.h"
 
 #endif  // BASE_TASK_POST_TASK_AND_REPLY_WITH_RESULT_INTERNAL_FORWARD_H_
diff --git a/base/sequenced_task_runner.cc b/base/task/sequenced_task_runner.cc
similarity index 100%
rename from base/sequenced_task_runner.cc
rename to base/task/sequenced_task_runner.cc
diff --git a/base/sequenced_task_runner.h b/base/task/sequenced_task_runner.h
similarity index 98%
rename from base/sequenced_task_runner.h
rename to base/task/sequenced_task_runner.h
index 5b92a89af4..d53b275b86 100644
--- a/base/sequenced_task_runner.h
+++ b/base/task/sequenced_task_runner.h
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef BASE_SEQUENCED_TASK_RUNNER_H_
-#define BASE_SEQUENCED_TASK_RUNNER_H_
+#ifndef BASE_TASK_SEQUENCED_TASK_RUNNER_H_
+#define BASE_TASK_SEQUENCED_TASK_RUNNER_H_
 
 #include <memory>
 
@@ -199,4 +199,4 @@ struct BASE_EXPORT OnTaskRunnerDeleter {
 
 }  // namespace base
 
-#endif  // BASE_SEQUENCED_TASK_RUNNER_H_
+#endif  // BASE_TASK_SEQUENCED_TASK_RUNNER_H_
diff --git a/base/task/sequenced_task_runner_forward.h b/base/task/sequenced_task_runner_forward.h
index 4ac3dbe401..31b1c9aad6 100644
--- a/base/task/sequenced_task_runner_forward.h
+++ b/base/task/sequenced_task_runner_forward.h
@@ -5,6 +5,6 @@
 #ifndef BASE_TASK_SEQUENCED_TASK_RUNNER_FORWARD_H_
 #define BASE_TASK_SEQUENCED_TASK_RUNNER_FORWARD_H_
 
-#include "base/sequenced_task_runner.h"
+#include "base/task/sequenced_task_runner.h"
 
 #endif  // BASE_TASK_SEQUENCED_TASK_RUNNER_FORWARD_H_
diff --git a/base/sequenced_task_runner_helpers.h b/base/task/sequenced_task_runner_helpers.h
similarity index 90%
rename from base/sequenced_task_runner_helpers.h
rename to base/task/sequenced_task_runner_helpers.h
index 4743db2080..a7fdd852c2 100644
--- a/base/sequenced_task_runner_helpers.h
+++ b/base/task/sequenced_task_runner_helpers.h
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef BASE_SEQUENCED_TASK_RUNNER_HELPERS_H_
-#define BASE_SEQUENCED_TASK_RUNNER_HELPERS_H_
+#ifndef BASE_TASK_SEQUENCED_TASK_RUNNER_HELPERS_H_
+#define BASE_TASK_SEQUENCED_TASK_RUNNER_HELPERS_H_
 
 #include <memory>
 
@@ -54,4 +54,4 @@ class ReleaseHelper {
 
 }  // namespace base
 
-#endif  // BASE_SEQUENCED_TASK_RUNNER_HELPERS_H_
+#endif  // BASE_TASK_SEQUENCED_TASK_RUNNER_HELPERS_H_
diff --git a/base/task/sequenced_task_runner_helpers_forward.h b/base/task/sequenced_task_runner_helpers_forward.h
index 06907c1c64..bd62ed7c03 100644
--- a/base/task/sequenced_task_runner_helpers_forward.h
+++ b/base/task/sequenced_task_runner_helpers_forward.h
@@ -5,6 +5,6 @@
 #ifndef BASE_TASK_SEQUENCED_TASK_RUNNER_HELPERS_FORWARD_H_
 #define BASE_TASK_SEQUENCED_TASK_RUNNER_HELPERS_FORWARD_H_
 
-#include "base/sequenced_task_runner_helpers.h"
+#include "base/task/sequenced_task_runner_helpers.h"
 
 #endif  // BASE_TASK_SEQUENCED_TASK_RUNNER_HELPERS_FORWARD_H_
diff --git a/base/sequenced_task_runner_unittest.cc b/base/task/sequenced_task_runner_unittest.cc
similarity index 100%
rename from base/sequenced_task_runner_unittest.cc
rename to base/task/sequenced_task_runner_unittest.cc
diff --git a/base/single_thread_task_runner.h b/base/task/single_thread_task_runner.h
similarity index 89%
rename from base/single_thread_task_runner.h
rename to base/task/single_thread_task_runner.h
index 9068946344..abe1abc89b 100644
--- a/base/single_thread_task_runner.h
+++ b/base/task/single_thread_task_runner.h
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef BASE_SINGLE_THREAD_TASK_RUNNER_H_
-#define BASE_SINGLE_THREAD_TASK_RUNNER_H_
+#ifndef BASE_TASK_SINGLE_THREAD_TASK_RUNNER_H_
+#define BASE_TASK_SINGLE_THREAD_TASK_RUNNER_H_
 
 #include "base/base_export.h"
 #include "base/task/sequenced_task_runner_forward.h"
@@ -33,4 +33,4 @@ class BASE_EXPORT SingleThreadTaskRunner : public SequencedTaskRunner {
 
 }  // namespace base
 
-#endif  // BASE_SINGLE_THREAD_TASK_RUNNER_H_
+#endif  // BASE_TASK_SINGLE_THREAD_TASK_RUNNER_H_
diff --git a/base/task/single_thread_task_runner_forward.h b/base/task/single_thread_task_runner_forward.h
index 89fed2ce75..365a530b70 100644
--- a/base/task/single_thread_task_runner_forward.h
+++ b/base/task/single_thread_task_runner_forward.h
@@ -5,6 +5,6 @@
 #ifndef BASE_TASK_SINGLE_THREAD_TASK_RUNNER_FORWARD_H_
 #define BASE_TASK_SINGLE_THREAD_TASK_RUNNER_FORWARD_H_
 
-#include "base/single_thread_task_runner.h"
+#include "base/task/single_thread_task_runner.h"
 
 #endif  // BASE_TASK_SINGLE_THREAD_TASK_RUNNER_FORWARD_H_
diff --git a/base/task_runner.cc b/base/task/task_runner.cc
similarity index 100%
rename from base/task_runner.cc
rename to base/task/task_runner.cc
diff --git a/base/task_runner.h b/base/task/task_runner.h
similarity index 98%
rename from base/task_runner.h
rename to base/task/task_runner.h
index cb568548ab..821ef98d3c 100644
--- a/base/task_runner.h
+++ b/base/task/task_runner.h
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef BASE_TASK_RUNNER_H_
-#define BASE_TASK_RUNNER_H_
+#ifndef BASE_TASK_TASK_RUNNER_H_
+#define BASE_TASK_TASK_RUNNER_H_
 
 #include <stddef.h>
 
@@ -180,4 +180,4 @@ struct BASE_EXPORT TaskRunnerTraits {
 
 }  // namespace base
 
-#endif  // BASE_TASK_RUNNER_H_
+#endif  // BASE_TASK_TASK_RUNNER_H_
diff --git a/base/task/task_runner_forward.h b/base/task/task_runner_forward.h
index c5b1a4174b..d5244aecee 100644
--- a/base/task/task_runner_forward.h
+++ b/base/task/task_runner_forward.h
@@ -5,6 +5,6 @@
 #ifndef BASE_TASK_TASK_RUNNER_FORWARD_H_
 #define BASE_TASK_TASK_RUNNER_FORWARD_H_
 
-#include "base/task_runner.h"
+#include "base/task/task_runner.h"
 
 #endif  // BASE_TASK_TASK_RUNNER_FORWARD_H_
diff --git a/base/task_runner_unittest.cc b/base/task/task_runner_unittest.cc
similarity index 100%
rename from base/task_runner_unittest.cc
rename to base/task/task_runner_unittest.cc
diff --git a/base/task_runner_util.h b/base/task/task_runner_util.h
similarity index 93%
rename from base/task_runner_util.h
rename to base/task/task_runner_util.h
index e8e245af2b..c3eb083a42 100644
--- a/base/task_runner_util.h
+++ b/base/task/task_runner_util.h
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef BASE_TASK_RUNNER_UTIL_H_
-#define BASE_TASK_RUNNER_UTIL_H_
+#ifndef BASE_TASK_TASK_RUNNER_UTIL_H_
+#define BASE_TASK_TASK_RUNNER_UTIL_H_
 
 #include <memory>
 #include <utility>
@@ -53,4 +53,4 @@ bool PostTaskAndReplyWithResult(TaskRunner* task_runner,
 
 }  // namespace base
 
-#endif  // BASE_TASK_RUNNER_UTIL_H_
+#endif  // BASE_TASK_TASK_RUNNER_UTIL_H_
diff --git a/base/task/task_runner_util_forward.h b/base/task/task_runner_util_forward.h
index 460233745f..3da6a3fc33 100644
--- a/base/task/task_runner_util_forward.h
+++ b/base/task/task_runner_util_forward.h
@@ -5,6 +5,6 @@
 #ifndef BASE_TASK_TASK_RUNNER_UTIL_FORWARD_H_
 #define BASE_TASK_TASK_RUNNER_UTIL_FORWARD_H_
 
-#include "base/task_runner_util.h"
+#include "base/task/task_runner_util.h"
 
 #endif  // BASE_TASK_TASK_RUNNER_UTIL_FORWARD_H_
diff --git a/base/task_runner_util_unittest.cc b/base/task/task_runner_util_unittest.cc
similarity index 100%
rename from base/task_runner_util_unittest.cc
rename to base/task/task_runner_util_unittest.cc
diff --git a/base/updateable_sequenced_task_runner.h b/base/task/updateable_sequenced_task_runner.h
similarity index 84%
rename from base/updateable_sequenced_task_runner.h
rename to base/task/updateable_sequenced_task_runner.h
index 5ab801d71c..66e2faebc4 100644
--- a/base/updateable_sequenced_task_runner.h
+++ b/base/task/updateable_sequenced_task_runner.h
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef BASE_UPDATEABLE_SEQUENCED_TASK_RUNNER_H_
-#define BASE_UPDATEABLE_SEQUENCED_TASK_RUNNER_H_
+#ifndef BASE_TASK_UPDATEABLE_SEQUENCED_TASK_RUNNER_H_
+#define BASE_TASK_UPDATEABLE_SEQUENCED_TASK_RUNNER_H_
 
 #include "base/task/sequenced_task_runner_forward.h"
 #include "base/task/task_traits.h"
@@ -27,4 +27,4 @@ class BASE_EXPORT UpdateableSequencedTaskRunner : public SequencedTaskRunner {
 
 }  // namespace base
 
-#endif  // BASE_UPDATEABLE_SEQUENCED_TASK_RUNNER_H_
+#endif  // BASE_TASK_UPDATEABLE_SEQUENCED_TASK_RUNNER_H_
diff --git a/base/task/updateable_sequenced_task_runner_forward.h b/base/task/updateable_sequenced_task_runner_forward.h
index f5e9469d9b..6c83bb540a 100644
--- a/base/task/updateable_sequenced_task_runner_forward.h
+++ b/base/task/updateable_sequenced_task_runner_forward.h
@@ -5,6 +5,6 @@
 #ifndef BASE_TASK_UPDATEABLE_SEQUENCED_TASK_RUNNER_FORWARD_H_
 #define BASE_TASK_UPDATEABLE_SEQUENCED_TASK_RUNNER_FORWARD_H_
 
-#include "base/updateable_sequenced_task_runner.h"
+#include "base/task/updateable_sequenced_task_runner.h"
 
 #endif  // BASE_TASK_UPDATEABLE_SEQUENCED_TASK_RUNNER_FORWARD_H_
-- 
2.35.1.1021.g381101b075-goog

